### PR TITLE
Update Model Store document

### PR DESCRIPTION
**Pull request by IsaacYangSLA**
Update document on UI changes.
Replace old screenshot with new UI.

### CODE REVIEW

File: docs/ModelStore.md [hunk 1]
```diff
@@ -27,12 +27,13 @@ The new page shows models available in model stores.
 
 ![List models](images/model-store-list.png)
 
-Hover the Note field to show complete text.
+Hover on Note will show the complete notes.
```
Comment (lukeyeager) [inline docs/ModelStore.md]: How about:

Hover **over "Note" to** show the complete notes.

File: docs/ModelStore.md [hunk 2]
```diff
@@ -27,12 +27,13 @@ The new page shows models available in model stores.
...
 Enter keyword in 'Filter list by' to limit results.
 Click 'Update model list' button will retrieve the latest model list (see limitation).
-Click 'Import' will import that model into DIGITS (may takes a few seconds, depends on network speed).
+Click 'Import' icon will import that model into DIGITS (may takes a few seconds, depends on network speed).
```
Comment (lukeyeager) [inline docs/ModelStore.md]: How about:

Click "Import" **to** import that model into DIGITS (may **take** a few seconds, **depending** on network speed).

File: docs/ModelStore.md [hunk 3]
```diff
@@ -69,4 +70,4 @@ The `license.txt` inside that subdirectory will be displayed when users clicking
 ## Limitation
 Some web server may limit frequent requests from the same machine to stop malicious activities.
 Therefore, DIGITS implemented a cache mechanism to reduce server-to-server communication.
-The button, 'Update model list', invalidates cache and retrieve meta data from all models.
+The button, 'Update model list', invalidates cache and retrieve meta data from all models from each Model Store.
```
Comment (lukeyeager) [inline docs/ModelStore.md]: How about:

invalidates **the** cache and **retrieves metadata** from

**Comment by IsaacYangSLA** [comment]
Close this one in favor of https://github.com/NVIDIA/DIGITS/pull/1346.

**Pull request closed without merging**